bin/emqx

```diff
@@ -159,9 +159,6 @@ usage() {
         echo "Print EMQX installation root dir"
         ;;
     eval)
-        echo "Evaluate an Erlang or Elixir expression in the EMQX node"
-        ;;
-    eval-erl)
```

Comment:
I think it's better to keep this option as-is, for backward compatibility as well.

Reply:
It's to be backward compatible with v4. On that version:
```
    eval)
        echo "Evaluate an Erlang expression in the EMQX node"
        ;;
```
So we're keeping this behavior (as suggested by @zmstone [here](https://github.com/emqx/emqx/pull/10263#issuecomment-1489777099)).

Reply:
I mean for `eval-erl`, it's possible someone is using that since it's released.

If it's fine to drop it since Elixir support is still kinda experimental, then just ignore me.  :smile_cat:

Reply:
I think its fine to drop it.